contracts/tutorial/Cargo.toml

```diff
@@ -38,12 +38,12 @@ backtraces = []
 
 [dependencies]
 provwasm-std = { path = "../../packages/bindings" }
-cosmwasm-std = { version = "1.0.0", default-features = false }
-cosmwasm-storage = { version = "1.0.0" }
+cosmwasm-std = { version = "1.1.2", default-features = false }
+cosmwasm-storage = { version = "1.1.2" }
 schemars = "0.8.3"
 serde = { version = "1.0.127", default-features = false, features = ["derive"] }
 thiserror = { version = "1.0.26" }
 
 [dev-dependencies]
 provwasm-mocks = { path = "../../packages/mocks" }
-cosmwasm-schema = { version = "1.0.0" }
+cosmwasm-schema = { version = "1.1.2" }
```

Comment:
Same comments as attrs/Cargo.toml.

Reply:
You are correct, I was delaying the schema refactoring, but is now included with this PR